### DISCUSSION TITLE
scEU-seq cell cycle problems with the animation of cell cycle

### DISCUSSION THREAD

**Discussion by xyfan1030** [aristoteleo/dynamo-release#468] [Q&A]
Hello, I found a problem in the scEU-seq cell cycle tutorial, when I was runing the last code "dyn.vf.VectorField(rpe1_kinetics, basis='RFP_GFP', map_topography=True, M=50)" following your steps, the error occurs that TypeError: object of type 'NoneType' has no len(). May I ask the solution?

**Comment by Xiaojieqiu** [comment]
Can you provide more details? e.g. the code you run that leads to your error

**Comment by Sichao25** [comment]
Hi, thank you very much for reporting this problem. We appreciate your contribution to the dynamo community. Right now VectorField function will raise the error if it can't find any fixed points. We will fix this problem very soon. 

One way to avoid this is to increase the number of samples for calculating the fixed points. You can try to tune parameter n like `dyn.vf.VectorField(rpe1_kinetics, basis='RFP_GFP', map_topography=True, M=50, n=100)`. Let me know if this solves your issue.
